src/main/java/org/broadinstitute/hellbender/cmdline/ArgumentCollection.java

```diff
@@ -13,4 +13,12 @@
 public @interface ArgumentCollection {
     /** Text that appears for this group of options in text describing usage of the command line program. */
     String doc() default "";
+
+    // The arguments in this collection are dependent on (and only enabled with the presence of)
+    // another argument. This may be used alone or together with dependsOnValue.
```

Comment:
It's a bit strange that these are at the ArgumentCollection level instead of on individual arguments.